slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/ChatUnfurlParamsIF.java

```diff
@@ -0,0 +1,31 @@
+package com.hubspot.slack.client.methods.params.chat;
+
+import java.net.URI;
+import java.util.Map;
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.Attachment;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ChatUnfurlParamsIF {
```

Comment:
I think this should extend `HasChannel`

Reply:
When I add `extends HasChannel` and annotate `getChannelId` with the `@Override`, I get a failing unit test:

```
[ERROR] Tests run: 24, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.052 s <<< FAILURE! - in com.hubspot.slack.client.HasChannelDeserializerTest
[ERROR] pojosDoDeserializeHasChannelsWithChannelField[class com.hubspot.slack.client.methods.params.chat.ChatUnfurlParams](com.hubspot.slack.client.HasChannelDeserializerTest)  Time elapsed: 0.008 s  <<< ERROR!
java.lang.IllegalStateException: Couldn't find a method with name 'builder' on class java.lang.Class
        at com.hubspot.slack.client.HasChannelDeserializerTest.pojosDoDeserializeHasChannelsWithChannelField(HasChannelDeserializerTest.java:27)
```

Weird because the `builder` method on the generated `ChatUnfurlParams` is there